### DISCUSSION TITLE
Oh My ZSH integration

### DISCUSSION THREAD

**Discussion by ZA0068** [marlonrichert/zsh-autocomplete#674] [Q&A] [answered]
## What do you want?
Hi, i want this to be integrated into Oh My ZSH shell: https://ohmyz.sh/

## Why do you want this?
This really makes my life easier since it helps not only finding the history easier but also finding directory easy as well and integration on Oh My ZSH will this make much more streamlined for oh my zsh users.

## Who else would benefit from this?
Everything, The possibilites are endless and hundreds and thausaids of hours will be saved.

## How should it work?
This should work as seemless when integrated as plugin into oh my zsh shell.

Given the following situation:
Because i use linux for my engineering course, i have to navigate through thousands of folders and this will hlp me a lot

When I perform the following steps:
similar to this:
git clone https://github.com/zsh-users/zsh-autosuggestions ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-autosuggestions

then you add this in the -zshrc:
plugins=( 
    # other plugins...
    zsh-autosuggestions
)

Then I expect the following to happen:
pressing down allows me to select directories
and pressing up will let me see histories without crashing.

**Comment by PythonCoderAS** [comment, marked as answer]
This already works for me with oh-my-zsh. I just ran this command and I could use it in plugins:
```shell
git clone https://github.com/marlonrichert/zsh-autocomplete.git  ${ZSH_CUSTOM:-~/.oh-my-zsh/custom}/plugins/zsh-autocomplete
```

**Comment by yngvark** [comment]
This should be part of the README's installation instructions.

**Reply by marlonrichert** [reply]
I disagree. It's not my responsibility to document OMZ. Tell OMZ to improve their instructions for installing plugins.

**Comment by nunojsa** [comment]
> This should be part of the README's installation instructions.

That was already purposed and rejected by the maintainer...

**Reply by marlonrichert** [reply]
I don't want to duplicate OMZ's instructions for installing plugins here.
